expression/builtin_json_vec.go

```diff
@@ -21,11 +21,30 @@ import (
 )
 
 func (b *builtinJSONDepthSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinJSONDepthSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETJson, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err := b.args[0].VecEvalJSON(b.ctx, input, buf); err != nil {
+		return err
+	}
+	result.ResizeInt64(n, false)
+	int64s := result.Int64s()
+	for i := 0; i < n; i++ {
+		if buf.IsNull(i) {
+			result.SetNull(i, true)
```

Comment:
We can use `MergeNulls` to speed up the Null operator.

Reply:
I have already used `MergeNulls`. Please review my code again.